File: framework/src/main/java/com/djangoogle/framework/widget/MarqueeTextView.java
package com.djangoogle.framework.widget;

import android.content.Context;
import android.support.v7.widget.AppCompatTextView;
import android.util.AttributeSet;

/**
 * 自定义支持跑马灯TextView
 * Created by Djangoogle on 2018/11/15 23:12 with Android Studio.
 */
public class MarqueeTextView extends AppCompatTextView {

	public MarqueeTextView(Context context) {
		super(context);
	}

	public MarqueeTextView(Context context, AttributeSet attrs) {
		super(context, attrs);
	}

	public MarqueeTextView(Context context, AttributeSet attrs, int defStyleAttr) {
		super(context, attrs, defStyleAttr);
	}

	@Override
	public boolean isFocused() {
		return true;
	}
}

File: framework/src/main/kotlin/com/djangoogle/framework/widget/MarqueeTextView.kt
package com.djangoogle.framework.widget

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

/**
 * 自定义支持跑马灯TextView
 * Created by Djangoogle on 2018/11/15 23:12 with Android Studio.
 */
class MarqueeTextView : AppCompatTextView {

	constructor(context: Context) : super(context, null)

	constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

	override fun isFocused(): Boolean {
		return true
	}
}
